cypress/integration/instance/InstanceEdit.ts

```diff
@@ -50,5 +50,67 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
             });
         });
     });
+
+    it.only("Switch from 'Webhook' to 'Ignore' Error handler", () => {
+      //Open 'Instance one' page
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId("Instance one", "PF4/TableRow")
+        .within(() => {
+          cy.get(
+            "td:first a[data-testid='tableInstances-linkInstance']"
+          ).click();
+        });
+
+      // Switch to error handling tab
+      cy.ouiaId("error-handling", "PF4/TabButton").click();
+
+      // Wait tab is loaded
+      cy.get(".pf-c-description-list__term").should("have.length", 5);
+      // Once tab is loaded start edit mode
+      cy.ouiaId("error-handling", "PF4/TabContent").within(() => {
+        cy.ouiaId("edit", "PF4/Button")
+          .should("have.attr", "aria-disabled")
+          .and("equal", "false");
+        cy.ouiaId("edit", "PF4/Button").click();
+      });
+
+      // Change error handling strategy to 'Ignore'
+      cy.ouiaId("error-handling-method-selector", "PF4/Select").within(() => {
+        cy.get("button").should("not.have.attr", "disabled");
+      });
+      cy.ouiaId("error-handling-method-selector", "PF4/Select").click();
+      cy.get("button.pf-c-select__menu-item", { timeout: 10000 })
+        .contains("Ignore")
+        .click();
+      cy.ouiaId("submit", "PF4/Button").click();
+
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId("Instance one", "PF4/TableRow")
+        .should("be.visible")
+        .within(() => {
+          //The user cannot go to the instance's details if it is not ready
```

Comment:
FYI this is going to change with https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/243

Reply:
I will remove that comment then probably